src/e2e/java/uk/gov/hmcts/reform/sscs/functional/sya/notifications/DocmosisWithGovNotifyLetterFunctionalTest.java

```diff
@@ -54,7 +54,7 @@ public void shouldSendDocmosisLetters(NotificationEventType notificationEventTyp
         return new Object[]{
             new Object[]{PROCESS_AUDIO_VIDEO, expectedNumberOfLettersIsThree},
             new Object[]{PROCESS_AUDIO_VIDEO_WELSH, expectedNumberOfLettersIsThree},
-            new Object[]{REQUEST_INFO_INCOMPLETE, expectedNumberOfLettersIsThree},
+            new Object[]{REQUEST_INFO_INCOMPLETE, expectedNumberOfLettersIsOne},
```

Comment:
Is it one, since we send a letter to one party at a time with this event?

Reply:
The whole point of this change is so we only send it to the party that they selected in the UI, so it will always be one